tests/topotests/bgp_conditional_advertisement/r1/bgpd.conf

```diff
@@ -0,0 +1,30 @@
+!
+ip prefix-list CUST seq 5 permit 10.139.224.0/20
+ip prefix-list DEFAULT seq 5 permit 0.0.0.0/0
+ip prefix-list PL1 seq 5 permit 1.1.1.1/32
```

Comment:
please use https://tools.ietf.org/html/rfc5737

Reply:
using 192.0.2.0/24, 198.51.100.0/24 and 203.0.113.0/24 is Ok OR
shall I use 192.0.2.0/26 for all routers. which is preferred one?

Reply:
Only external IP addresses need to be changed.

Reply:
ok